scripts/migrate-patterns.sh

```
@@ -0,0 +1,107 @@
+  number=$(echo "$issue" | jq -r .number)
+  title=$(echo "$issue" | jq -r .title)
+  body=$(echo "$issue" | jq -r .body)
+
+  # 跳过已迁移的
+  if echo "$body" | grep -q "## 模式"; then
+    SKIPPED=$((SKIPPED + 1))
+    continue
+  fi
+
+  # 从 title 提取 pattern id（格式：pattern: xxx）
+  id=$(echo "$title" | sed 's/^pattern: //')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)